keyring.go

```diff
@@ -290,6 +289,19 @@ func (k KeyRing) VerifyJSONs(ctx context.Context, requests []VerifyJSONRequest)
 		}
 	}
 
+	// We for some reason failed to fetch keys for some servers
+	if len(keyRequests) > 0 {
+		requestedServers := make([]string, 0, len(keyRequests))
+		for reqs := range keyRequests {
+			requestedServers = append(requestedServers, string(reqs.ServerName))
+		}
+
+		logger.WithFields(logrus.Fields{
+			"servers":  requestedServers,
+			"fetchers": len(k.KeyFetchers),
+		}).Warn("failed to fetch keys for some servers")
```

Comment:
Without stating which servers this isn't very helpful.

Reply:
`requestedServers` is the list of the servers we failed to fetch keys for.

Reply:
Ah, that wasn't clear. LGTM then!